services/github_service.py

```diff
@@ -1263,3 +1263,54 @@ def get_old_poc_repositories(self) -> list:
                 old_poc_repositories[repo] = age
 
         return old_poc_repositories
+
+    @retries_github_rate_limit_exception_at_next_reset_once
+    def get_user_removal_events(self, since_date: str, actor: str) -> list:
+        logging.info(f"Getting audit log entries for users removed by {actor} since {since_date}")
+        today = datetime.now()
+        query_string = f"action:org.remove_member actor:{actor} created:{since_date}..{today.strftime('%Y-%m-%d')}"
+
+        query = """
+            query($organisation_name: String!, $query_string: String!, $cursor: String) {
+                organization(login: $organisation_name) {
+                    auditLog(
+                        first: 100
```

Comment:
I take it there is unlikely to be more than 100 audit log entries for users removed by each actor?